### DISCUSSION TITLE
Cant' use any version after 1.2.3

### DISCUSSION THREAD

**Discussion by rop12770** [rustdesk/rustdesk#9254] [Q&A] [answered]
Hey guys

I'm running Kubuntu 24.04 6.8.0-41-generic (64-bits), and if I update rustdesk to any version after 1.2.3, after reboot, rustdesk always open, close, open, close until I kill the task, which is a pain to do.
All versions were tested and the same issues...
Any ideas?
Thanks

**Comment by rustdesk** [comment]
Any log? or more information rather than only `open, close, open, close`.

**Comment by rop12770** [comment]
I have a couple of videos:
https://photos.app.goo.gl/C8RNuNg7EUHkvKzHA
https://photos.app.goo.gl/Tgboo8uzJUsZcUvU6

**Reply by rustdesk** [reply]
>  until I kill the task

Which task?

**Reply by rustdesk** [reply]
It seems `--server` can not start on your machine. Could you paste logs in `/root/.local/share/logs/RustDesk/server/` or `~/.local/share/logs/RustDesk/server/`? also show `ls` please.

**Reply by rop12770** [reply]
Hi

Here you go: 
![image](https://github.com/user-attachments/assets/bbf1ce97-8abe-4c6b-9be1-26a98c15fc82)
[server Logs.zip](https://github.com/user-attachments/files/16887433/server.Logs.zip)

Thanks

**Reply by rustdesk** [reply]
@21pages follow it up.

**Reply by rustdesk** [reply]
Can you `ls -ls /tmp/RustDesk/ipc`?

**Reply by rop12770** [reply]
Is this what you want?
![image](https://github.com/user-attachments/assets/49bd1ba7-872b-4f3b-a8fc-654fb0be05fc)

**Reply by rustdesk** [reply]
Could you open RustDesk GUI later after reboot?

**Reply by rop12770** [reply]
After update?

**Comment by rustdesk** [comment]
We have a fix https://github.com/rustdesk/rustdesk/pull/9264, you can try later once the nightly build is ready.

**Comment by rustdesk** [comment]
Please try out the latest nightly, https://github.com/rustdesk/rustdesk/releases/tag/nightly
We refactored the code, there won't be close/open loop.

**Reply by rop12770** [reply]
I have updated. I will test it and update you.
Thanks

**Comment by rop12770** [comment, marked as answer]
Hey guys

Just to let you know that the fix worked.
Thanks
